BluetraceUtils.swift

```diff
@@ -41,30 +42,4 @@ class BluetraceUtils {
             return "unknown"
         }
     }
-
-    static func removeData21DaysOld() {
```

Comment:
Looks like this functionality is moved to the `AppDelegate.clearOldDataInContext()` method in COVIDsafe using `Encounter.fetchOldEncounters()`. I'll add AppDelegate.swift to this PR as well.